modules/ui/map3d_viewer.js

```diff
@@ -251,6 +254,39 @@ export function uiMap3dViewer(context) {
           .duration(200)
           .style('opacity', '1')
           .on('end', () => redraw());
+        urlhash.setParam('map3d', 'true');
+        }
+    }
+
+    function _hashchange(){
+      // _isHidden = !_isHidden;
+      let _isHidden = context.systems.urlhash?.getParam('map3d');
```

Comment:
I'll be cryptic here and say that I don't think that this line is doing what you think it should be doing. 😈 

hint: Don't `let` the assignment fool you.

Reply:
So, I'm now toggling the global `_isHidden` variabnle from inside `toggle()`, however, I still have the same bug I from here https://github.com/facebook/Rapid/pull/1186#issuecomment-1790968469

I'll share some screenshots and code to show you